File: src/main/java/com/pauljoda/modularsystems/storage/container/ContainerStorageRemote.java
package com.pauljoda.modularsystems.storage.container;

import com.teambr.bookshelf.inventory.BaseContainer;
import net.minecraft.inventory.IInventory;

/**
 * Modular-Systems
 * Created by Dyonovan on 01/08/15
 */
public class ContainerStorageRemote extends BaseContainer {
    public ContainerStorageRemote(IInventory playerInventory, IInventory ownerInventory) {
        super(playerInventory, ownerInventory);
        addSlotToContainer(new RestrictedSlot(ownerInventory, 0, 80, 35));
        addSlotToContainer(new RestrictedSlot(ownerInventory, 1, 80, 60));
        addPlayerInventorySlots(84);
    }
}

File: src/main/scala/com/teambr/modularsystems/storage/container/ContainerStorageRemote.scala
package com.teambr.modularsystems.storage.container

import com.teambr.bookshelf.common.container.BaseContainer
import com.teambr.modularsystems.storage.tiles.TileStorageRemote
import net.minecraft.entity.player.InventoryPlayer

/**
 * This file was created for Modular-Systems
 *
 * Modular-Systems is licensed under the
 * Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International License:
 * http://creativecommons.org/licenses/by-nc-sa/4.0/
 *
 * @author Dyonovan
 * @since August 10, 2015
 */
class ContainerStorageRemote(playerInventory: InventoryPlayer, tile: TileStorageRemote)
        extends BaseContainer(playerInventory, tile) {

    addSlotToContainer(new RestrictedSlot(tile, 0, 56, 35))
    addSlotToContainer(new RestrictedSlot(tile, 1, 116, 35))
    addPlayerInventorySlots(84)


}
